config/manifests-crd/bases/recipe.clusterserviceversion.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
Is this file generated by some tool?

Reply:
This is generated by controller-gen but it should be committed as it has provider info and description that is lost on every generation if not committed.